gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/spi/DefaultResourceManagerRpc.java

```diff
@@ -1,9 +1,5 @@
-package com.google.gcloud.spi;
+package com.google.gcloud.resourcemanager.spi;
 
-import static com.google.gcloud.spi.ResourceManagerRpc.Option.FIELDS;
```

Comment:
Nit: we could keep this static import and remove the "Option." additions below, right?